gulpfile.js

```diff
@@ -57,31 +71,61 @@ const cleanDist = (cb) => {
 };
 cleanDist.displayName = '🧹 cleaning up dist path';
 
+/**
+ * Copy Sass layout files from design-system-tokens to theme styles folder
+ */
+
+const copySassLayoutTokens = (cb) => {
+  const sassLayoutTokensFiles = `${tokensPackageFiles}/css-vars/${theme()}-layout-tokens.scss`;
+
+  gulp
+    .src(sassLayoutTokensFiles)
+    .pipe(rename('_layout.scss'))
```

Comment:
I found it! What will allow us to avoid copying this file is [the at-forward rule](https://sass-lang.com/documentation/at-rules/forward/). If we replace our `src/styles/_layout.scss` file contents with the following, we'll be able to remove this gulp task:

```scss
@forward "../../../design-system-tokens/dist/css-vars/[THEME_NAME]-layout-tokens.scss";
```